### DISCUSSION TITLE
Presence detection/Intelligent eye and comfort mode

### DISCUSSION THREAD

**Discussion by kjoconnor** [revk/xESP32-Faikout#330] [Ideas]
On my model (CTXS07WVJU9) it supports a couple features I don't see in the web dashboard nor mentioned in my searching through the docs:

* "Intelligent Eye" - presence detection, gimmicky but may be useful if values can be read, or at least the setting toggled
* Comfort mode - tries to control the swing level to be least bothersome

Shown here from the [manual](https://www.daikinac.com/content/assets/DOC/OperationManuals/2017/CTXS07-FTXS09-12-Operation-Manual.pdf):

<img width="581" alt="image" src="https://github.com/revk/ESP32-Faikin/assets/731233/a5a94c8e-1ab9-4d3b-b34d-aed5ee0e4f40">

**Comment by revk** [comment]
We usually handle comfort.

But I am not sure the presence is in S21.

If you can work out the S21 messages for what you need, I can add.
